src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/ApplicationInsightsSinkTests.cs

```diff
@@ -65,7 +65,7 @@ protected ILoggerFactory CreateLoggerFactory(Action<LoggerConfiguration> configu
 
         protected async Task RetryAssertUntilTelemetryShouldBeAvailableAsync(Func<Task> assertion)
```

Comment:
Can we revert these changes?

Reply:
Yes we can; I just thought it was overkill to query AppInsights every second.

Reply:
This seems like a long wait though?
https://github.com/arcus-azure/arcus.observability/pull/181/files/6c7cfe05d76513b14edd981cb8fe4602432ffbe1#diff-27f08c3b8059324291530235f56cd9bb92ef433ef8842cd03175f58a2213f06bR68

Reply:
That's the max timeout.  So if it succeeds before that time, it won't wait that long

Reply:
Put it back to the original (7 minutes).  Kept the 3 seconds delay though instead of the 1 sec delay between app insights queries